terms/noiseindatasets_is.tex

```diff
@@ -0,0 +1,100 @@
+
+\begin{figure}
+  \includegraphics{graphics/noise_in_datasets/noise_imputation.png}
+  \caption{The approach adopted by \citet{zhu2004class} to filter and correct attribute noise.}
+  \label{fig:noise_imputation}
+\end{figure}
+
+Figure~\ref{fig:noise_imputation} depicts the approach taken by \citet{zhu2004class} to correct attribute noise\index{noise in datasets!attribute noise}, where for each attribute a strong correlation with other attributes is sought upon which noisy instances can be predicted (through a learning model) and corrected.  If a correlation is not found, corrections are based on other methods such as \textit{clustering} or \textit{k-nearest neighbour}.
+
+The choice of noise filtering method should be based on the task at hand \citep{frenay2014classification}. The effectiveness of the noise filter\index{noise in datasets!filter} should be evaluated on a dataset that is degraded with artificial noise\index{noise in datasets!artificial}.  The filtering precision can then be calculated through the number of correct instances that are filtered\index{noise in datasets!Type I errors} (\textit{Type I errors} - Equation~\ref{eq:noise_type1}) and the number of incorrect instances that are not filtered (\textit{Type II errors} - Equation~\ref{eq:noise_type2})\index{noise in datasets!Type II errors}.  These are calculated as follows: 
+
+\begin{equation}\label{eq:noise_type1}
+	ER\textsubscript{1} = \frac{\textnormal{\# of correctly labelled instances which are removed}}{\textnormal{\# of correctly labelled instances}}
+\end{equation}
+
+\begin{equation}\label{eq:noise_type2}
+	ER\textsubscript{2} = \frac{\textnormal{\# of mislabelled instances which are not removed}}{\textnormal{\# of mislabelled instances}}
+\end{equation}
+
+Consequently, the \textit{Noise Elimination Precision} (NER)\index{noise in datasets!Noise Elimination Precision} can be calculated through Equation~\ref{eq:noise_ner} \citep{frenay2014classification}.
+
+\begin{equation}\label{eq:noise_ner}
+	NER = \frac{\textnormal{\# of mislabelled instances which are removed}}{\textnormal{\# of removed instances}}
+\end{equation}
+
+\subsection{Noise robust models}\label{sec:noise_robust_models}\index{noise in datasets!noise robust models}
+
+No learning algorithm is immune to noise but some algorithms perform better than others in the presence of noise \citep{frenay2014comprehensive}.  \citet{kalapanidas2003machine} studied the noise sensitivity of ten different machine learning algorithms against various levels of artificially induced noise.  The results show that classifiers are much more noise tolerant than regressors.  The \textit{linear regression} algorithm proved to be the regressor least sensitive to noise, whilst the \textit{decision table classifier} proved to be the best performer overall.  A similar study \citep{nettleton2010study} found the \textit{naïve bayes} algorithm to be the most robust to noise.
+
```

Comment:
Did you look at the PDF?  Your dotted ï in Naive has to be a special LaTeX character otherwise it looks broken.